src/lib/dnssd/Advertiser_ImplMinimalMdns.cpp

```diff
@@ -621,6 +622,7 @@ CHIP_ERROR AdvertiserMinMdns::Advertise(const CommissionAdvertisingParameters &
 
     // Advertise the records we just added as required by RFC 6762.
     // TODO - Don't announce records that haven't been updated.
+    DeviceLayer::ConfigurationMgr().StartAdvertisement();
```

Comment:
I think we need this call to be in Dnssd.cpp rather than here since this method can be called multiple times without re-entry into commissioning mode. We want to be careful to not increment multiple times within a single commissioning (or extended discovery) session.